test/shmemx/Makefile.am

```diff
@@ -31,23 +25,27 @@ check_PROGRAMS += \
 	shmem_ct
 endif
 
-if SHMEMX_TESTS
-
 check_PROGRAMS += \
 	perf_counter \
 	fadd_nbi \
 	atomic_nbi \
 	put_signal \
-	put_signal_nbi
+	put_signal_nbi \
+	shmemx_wait_until_all \
+	shmemx_wait_until_any \
+	shmemx_wait_until_some \
+	shmemx_test_all \
+	shmemx_test_any \
+	shmemx_test_some \
+	c11_test_shmemx_wait_until \
+	c11_test_shmemx_test
```

Comment:
The move of this stuff from above down to here is the only change I would keep from this PR.

Reply:
No problem - this was the original intent of the PR, but I was confused about why some tests are built in `shmemx` even with `SHMEMX_TESTS` turned off... hence the proposed change.